scm/driver/github/webhook.go

```diff
@@ -356,6 +391,35 @@ func convertPullRequestHook(src *pullRequestHook) *scm.PullRequestHook {
 	}
 }
 
+func convertPullRequestReviewCommentHook(src *pullRequestReviewCommentHook) *scm.PullRequestCommentHook {
```

Comment:
I believe this should return a `ReviewCommentHook`. The distinction is that a pull request comment is when you comment on the issue (although github treats pull requests the same as issues, other providers do not, so we abstract). The review comment is when you comment on code. The `ReviewCommentHook` also stores a `Review` information such as file name and line number.